hack/build.sh

```diff
@@ -92,12 +92,13 @@ for dir in ${dirs}; do
   fi
 
   if [[ -v TEST_MODE ]]; then
-    VERSION=$dir IMAGE_NAME=${IMAGE_NAME} test/run
-
+    TEST_DIR=../hack
+    pushd $TEST_DIR && make $TEST_CASE VERSION=$dir IMAGE_NAME=${IMAGE_NAME}
```

Comment:
Why not? Since we need a way to actually run `make` on the tests, we can't go through the soft link. Well, we could just run the test directly but that would defeat the purpose of the Makefile. Or should I just postpone introduction of the Makefile to a later patch?